tests/core.py

```diff
@@ -2030,26 +2030,24 @@ class SSHHookTest(unittest.TestCase):
     def setUp(self):
         configuration.load_test_config()
         from airflow.contrib.hooks.ssh_hook import SSHHook
-        self.hook = SSHHook()
+        self.hook = SSHHook(ssh_conn_id='ssh_default')
         self.hook.no_host_key_check = True
 
-    def test_remote_cmd(self):
-        output = self.hook.check_output(["echo", "-n", "airflow"])
-        self.assertEqual(output, b"airflow")
+    def test_ssh_connection(self):
```

Comment:
Please move these tests out of core and in their own file. I know you are just updating them, core is really I need of refactoring and this is a way to start and not to make it one big thing.